.github/workflows/ci_engine.yml

```diff
@@ -66,14 +66,14 @@ jobs:
       - name: Get wren binary
         uses: actions/download-artifact@v3
         with:
-          name: wren-binary
-          path: ci-bin/wren/
+          name: manager-binary
+          path: release/linux/amd64/
 
       - name: Get Sparrow binary
         uses: actions/download-artifact@v3
         with:
           name: engine-debug
-          path: ci-bin/engine/
+          path: release/linux/amd64/
```

Comment:
i realize it is a bit odd copying a debug build to a release path, but this works.  Maybe in the future we can drop the `release` copy path in `Dockerfile.release`